addOns/pscanrulesBeta/CHANGELOG.md

```diff
@@ -4,7 +4,14 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+### Changed
+- Now targeting ZAP 2.10.
+- The following scan rules now support Custom Page definitions:
+  - Insecure Form Load
+  - Insecure Form Post
+  - User Controlled Charset
+  - User COntrolled HTML Attribute
```

Comment:
There's a typo in controlled (same for following line).

Reply:
Good catch, fix coming.